src/renderers/dom/client/ReactMount.js

```diff
@@ -382,11 +382,11 @@ var ReactMount = {
       'ReactDOM.render(): Invalid component element.%s',
       (
         typeof nextElement === 'string' ?
-          ' Instead of passing an element string, make sure to instantiate ' +
-          'it by passing it to React.createElement.' :
+          ' Instead of passing an element like \'div\', pass ' +
```

Comment:
This message seems to imply that what the user passed in was an element.  'div' is not an element, it is a string.  Perhaps "Instead of passing a string like 'div', ..."